src/2025h1/all-hands.md

```diff
@@ -0,0 +1,92 @@
+# Rust All-Hands 2025!
+
+| Metadata |                       |
+| -------- | --------------------- |
+| Owner(s) | Mara                  |
+| Teams    | [leadership-council]  |
+| Status   | Proposed              |
+
+## Summary
+
+Organise another Rust All-Hands in 2025!
+
+## Motivation
+
+It's been too long since we've had a Rust All-Hands. Time to bring it back!
+
+### The status quo
+
+Previous Rust All-Hands events were very useful and succesful, but after the changes at Mozilla, we haven't had a Rust All-Hands in six years.
+Meanwhile, the project has grown a lot, making it much harder and expensive to organise a new Rust All-Hands.
+
+A few months ago, Mara proposed having a new Rust All-Hands around the 10th birthday of Rust 1.0, at [RustWeek 2025](https://rustweek.org) by [RustNL](https://rustnl.org) in the Netherlands.
+Both RustNL and the Rust Leadership Council agreed.
+
+See https://blog.rust-lang.org/inside-rust/2024/09/02/all-hands.html
+
+### The next 6 months
+
+- Prepare the all-hands and everything around it.
+- Have a social and informal "pre-all hands day" on Rust's 10th birtday: May 15, 2025.
+- Have a two-day all-hands on May 16 and May 17, 2025.
+
+### The "shiny future" we are working towards
+
+The immediate goal is a very succesful and productive Rust All-Hands 2025.
+Hopefully, this will be the first step twoards the larger goal of having regular Rust All-Hands again.
```

Comment:
Personally, I don't think we will ever reach a point where a yearly All-Hands can work out logistically or financially. I think *regular* All-Hands would be good (maybe every 3 years, matching the edition), supplemented with unconfs+team meetups in between.

Reply:
Added a note that the frequency is an open question.